src/org/feedhenry/GitHubUtils.groovy

```diff
@@ -11,30 +11,33 @@ def ghApiRequest(String endpoint, String httpMethod = 'GET', String requestBody
             contentType: 'APPLICATION_JSON',
             requestBody: requestBody,
             customHeaders: customHeaders,
-            url: url
+            url: url,
+            validResponseCodes: '100:399,404'
```

Comment:
Probably not, i had meant to have this validResponsCodes string be passed in optionally like customHeaders